package.json

```diff
@@ -42,16 +42,16 @@
     "@bem-react/pack": "1.5.1",
     "@commitlint/cli": "13.2.1",
     "@commitlint/config-conventional": "13.2.0",
-    "@storybook/addon-essentials": "6.3.12",
-    "@storybook/react": "6.3.12",
-    "@testing-library/jest-dom": "5.14.1",
-    "@testing-library/react": "12.1.2",
+    "@storybook/addon-essentials": "6.5.10",
+    "@storybook/react": "6.5.10",
+    "@testing-library/jest-dom": "^5.16.5",
```

Comment:
i think, we should not use ^

Reply:
here agree — that one was unintentional

but for `@types/*` I think we could use `^`